modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -402,7 +402,8 @@ public CompletableFuture<Integer> createZone(DistributionZoneConfigurationParame
                         }
 
                         if (distributionZoneCfg.dataStorageChangeConsumer() == null) {
-                            zoneChange.changeDataStorage(ch -> ch.convert(zonesConfiguration.defaultDataStorage().value()));
+                            // TODO: IGNITE-20114 Delete when switching to the catalog
+                            zoneChange.changeDataStorage(ch -> ch.convert("aipersist"));
```

Comment:
Let's hardcode it in a constant, not in a literal :)

Reply:
That code in the neighboring [PR](https://github.com/apache/ignite-3/pull/2415) disappears, I don’t think that a constant is needed here.